{
  try {
    template.request(url,null);
    fail("Should have thrown a timeout exception");
  }
 catch (  Exception e) {
    ExchangeTimedOutException cause=assertIsInstanceOf(ExchangeTimedOutException.class,e.getCause());
    assertEquals(2000,cause.getTimeout());
  }
}
